WalletWasabi.Gui/Controls/WalletExplorer/TransactionViewModel.cs

```diff
@@ -55,27 +56,27 @@ public TransactionViewModel(TransactionInfo model)
 				});
 		}
 
-		private TransactionInfo Model { get; }
+		private TransactionDetailsViewModel TransactionDetails { get; }
 
 		public ReactiveCommand<Unit, Unit> CopyTransactionId { get; }
 
 		public ReactiveCommand<Unit, Unit> OpenTransactionInfo { get; }
 
-		public string DateTime => Model.DateTime.ToString("yyyy-MM-dd HH:mm", CultureInfo.InvariantCulture);
+		public string DateTime => TransactionDetails.DateTime.ToString("yyyy-MM-dd HH:mm", CultureInfo.InvariantCulture);
 
-		public bool Confirmed => Model.Confirmed;
+		public bool Confirmed => TransactionDetails.Confirmed;
```

Comment:
This should be removed.

Note that you need to make changes in https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Gui/Controls/WalletExplorer/HistoryTabView.xaml#L55
https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Gui/Controls/WalletExplorer/CoinListView.xaml#L118